From 2d036e6886fb68707e861a6e5f75a6088d149832 Mon Sep 17 00:00:00 2001
From: Michael <mcktr55@gmail.com>
Date: Tue, 26 Sep 2017 22:01:07 +0200
Subject: [PATCH] Fixed missing include statement in unit test

This adds the missing iostream include to the notification unit test.

refs #5613
---
 test/icinga-notification.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/icinga-notification.cpp b/test/icinga-notification.cpp
index 8db93fa50..97f7838a1 100644
--- a/test/icinga-notification.cpp
+++ b/test/icinga-notification.cpp
@@ -19,6 +19,7 @@
 
 #include "icinga/notification.hpp"
 #include <BoostTestTargetConfig.h>
+#include <iostream>
 
 using namespace icinga;
 
